### ISSUE TITLE
Upcasting Spells does not increase Damage Rolled

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#171]
In GitLab by @akrigline on Dec 14, 2020, 07:57

foundry core 0.7.8
dnd5e 1.2.0
midi-qol 0.3.35

## Repro Steps:
0. Cleric Character imported from ddb-importer before 1.2.0, has Guiding Bolt and higher level slots.
1. Cast Guiding Bolt from the Spellbook tab of the actor by clicking on the Item Image
2. Drop down the spell level, and select something higher than level 1.
3. Observe damage rolled

### Expected
4d6 + 1d6 per upcast level

### Actual
4d6

Other active possibly relevant modules:
- dae 0.2.27
- dnd5e helpers 1.8.0
- compact dnd beyond-like character sheet

Active Midi Settings:
```
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/d291f4bf9fee6afa475ecfc41013a4779224a5304ad843c4b27c6353c32707e5/image.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/cda4d1bd80e5fe354acbe4a06927a36e08fd12005bca48c27475643e48586bd1/image.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/1ebe14cedf214de46206f087ca72abb7a2371d1e0bdf4c927b68df022ec3050b/image.png)
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/480e31b8547d1488809d2e376e9f96181389a71cd4e5b531b702d2069953309f/image.png)
```

**Comment by tposney** [comment]
In GitLab by @akrigline on Dec 14, 2020, 08:45

I see that you have opened an issue in the dnd5e system about this, linking here for paper trail.

https://gitlab.com/foundrynet/dnd5e/-/issues/881

**Comment by tposney** [comment]
I have a work around that I'll release today, so users wont have to wait on any dnd5e changes.

**Comment by tposney** [comment]
In GitLab by @Alatreon on Dec 21, 2020, 21:45

As it is now, it consumes the correct spell slot, but still treats the damage as if it wasn't upcast at all.

**Comment by tposney** [comment]
Thanks. I screwed this up, it will get fixed in 0.3.41
